py3status/modules/getjson.py

```diff
@@ -68,28 +65,17 @@ class Py3status:
     def getjson(self):
         """
         """
-        response = {
-            'cached_until': self.py3.time_in(self.cache_timeout),
-        }
-
         try:
-            resp = self.py3.request(self.url, timeout=self.timeout)
-            status = resp.status_code == 200
-            resp = resp.json()
+            getjson = self.py3.request(self.url, timeout=self.timeout).json()
+            getjson = self.py3.flatten_dict(
+                getjson, delimiter=self.delimiter, intermediates=True)
         except self.py3.RequestException:
-            resp = None
-            status = False
-
-        if status:
-            response['full_text'] = self.py3.safe_format(
-                self.format,
-                self.py3.flatten_dict(
-                    resp, delimiter=self.delimiter, intermediates=True
-                )
-            )
-        else:
-            response['full_text'] = ''
-        return response
+            getjson = None
```

Comment:
This is a good scenario. We don't know what `{placeholder}` we can use with `getjson`.

How do you suggest we `''` the unknown placeholders... but still display the format? I'm guessing same for-loop `empty_status` in deadbeef.

If I use `format = "GETJSON: {moo}"`, should I get `GETJSON:` or `GETJSON: {moo}` or `''`?

`<nothing>` <- what we show currently -- Goes against my `GETJSON: ` format.
`GETJSON: {moo}` <- after this PR placeholders show as 'missing'.  Right. We should update our format to `GETJSON[: {moo}]` then... but this shows us that the URL is not available right now... or something unexpected happened.

`info:` <- I like this we make all placeholders '' so they get hidden`.... I think this may be harder to make things consistent between modules. Easier to just pass things as-is... if things may not be available sometimes, then we update format with your weaponized `the-formatter` gun.

I think using `[{placeholder}]` is nice, makes sense, hints people to learn a bit more about the formatter... than adding for-loop `empty_status` in all modules. Idk.

I think it's good to display `GETJSON: {moo} 1232.34`... Shows us that we need to fix `{moo}` placeholder somehow.... than `GETJSON: 1234.43` then people might not know about this broken/wrong input/typos placeholders. Again, IDK.